pkg/oc/graph/kubegraph/nodes/nodes.go

```diff
@@ -259,7 +259,7 @@ func EnsureDeploymentNode(g osgraph.MutableUniqueGraph, deployment *extensions.D
 	node := osgraph.EnsureUnique(g,
 		nodeName,
 		func(node osgraph.Node) graph.Node {
-			return &DeploymentNode{Node: node, Deployment: deployment}
+			return &DeploymentNode{Node: node, Deployment: deployment, IsFound: true}
```

Comment:
why are we setting this to IsFound? I thought that only got set when we verified its existence in the API

Reply:
We [set it to `false` in `FindOrCreateSynthetic...Node`](https://github.com/openshift/origin/blob/master/pkg/oc/graph/kubegraph/nodes/nodes.go#L103), when a synthetic node is created if a resource is not able to be retrieved / found. 

We appear to set IsFound to `true` here [in `Ensure...Node` funcs](https://github.com/openshift/origin/blob/master/pkg/oc/graph/kubegraph/nodes/nodes.go#L124)

Reply:
not in &StatefulSetNode{node, statefulSet, false}...

Reply:
> not in &StatefulSetNode{node, statefulSet, false}...

That appears to be a mistake I made in this PR:
https://github.com/openshift/origin/pull/18579/files#diff-5b718bf0b0ba776d85871d51d87dd74a

Ensure...Node functions are [called from projectstatus.go](https://github.com/openshift/origin/blob/master/pkg/oc/cli/describe/projectstatus.go#L1612) for [every node that is successfully loaded from the server](https://github.com/openshift/origin/blob/master/pkg/oc/cli/describe/projectstatus.go#L1600).

I have updated this PR to correct this.

Additionally, it appears that a `ScalingEdgeKind` [was not being added between hpa nodes and replicaSet nodes](https://github.com/openshift/origin/blob/master/pkg/oc/graph/kubegraph/edges.go#L259). I have updated this PR to address this as well.